controller/plugins/controller/kubejobs/plugin.py

```diff
@@ -88,3 +90,31 @@ def stop_application_scaling(self):
 
     def status(self):
         return self.alarm.status()
+
+    def validate(self, data):
+        data_model = {
+            "actuation_size": int,
+            "actuator": six.string_types,
+            "check_interval": int,
+            "max_rep": int,
+            "metric_source": six.string_types,
+            "min_rep": int,
+            "trigger_down": int,
+            "trigger_up": int
+        }
+
+        for key in data_model.keys():
```

Comment:
calling `data_model.keys()` is unnecessary and expensive. change it to `for key in data_model`.